rules/forbidden_actions.tex

```diff
@@ -106,7 +106,7 @@ \subsubsection{Penalty Area During Penalty Kick}
 
 \subsection{Forbidden Motion}
 Robots that begin moving their legs or locomote in any fashion before the specified state in the subsections
- will be penalized \textit{in place} on the field for \qty{15}{\second} in \textit{playing}.
+ will be penalized \textit{in place} on the field until the penalty is over for \qty{15}{\second} in the \textit{playing} state.
```

Comment:
That you are penalized "until the penalty is over" should be clear and does not resolve the issue. 
However, I would leave the sentence as before and add a note like: 

It's important to note that the penalty duration only counts down during the playing state.